ocf_data_sampler/load/utils.py

```diff
@@ -26,4 +26,12 @@ def make_spatial_coords_increasing(ds: xr.Dataset, x_coord: str, y_coord: str) -
     assert (ds[x_coord].diff(dim=x_coord) > 0).all()
     assert (ds[y_coord].diff(dim=y_coord) > 0).all()
 
-    return ds
\ No newline at end of file
+    return ds
+
+
+def underlying_array(ds: xr.Dataset, datavar: str | None = None) -> xr.DataArray:
+    if datavar:
+        return ds[datavar]
+    datavars = list(ds.var())
+    assert len(datavars) == 1, "Cannot open as xr.DataArray: dataset contains multiple variables"
+    return ds[datavars[0]]
```

Comment:
should this function be called change_xr_dataset_to_xr_data_array, or something like this?